basicdataset/src/test/resources/mlrepo/dataset/nlp/ai/djl/basicdataset/goemotions/metadata.json

```diff
@@ -0,0 +1,39 @@
+{
+  "metadataVersion": "0.2",
+  "resourceType": "dataset",
+  "application": "nlp",
+  "groupId": "ai.djl.basicdataset",
+  "artifactId": "goemotions",
+  "name": "goemotions",
+  "description": "GoEmotions contains 58k carefully curated Reddit comments labeled for 27 emotion categories or Neutral.",
+  "website": "https://github.com/google-research/google-research/tree/master/goemotions",
+  "licenses": {
+    "license": {
```

Comment:
I updated this to include the license. If you haven't done much with them, licenses are important for knowing what open source projects you are legally allowed to use. It was mentioned at https://github.com/google-research/google-research/blob/master/README.md if you just couldn't find it.